### DISCUSSION TITLE
Ender3 V2 Locked on firmware version 2.1.3 ( 4.2.2 )

### DISCUSSION THREAD

**Discussion by Dieter-hamer** [mriscoc/Ender3V2S1#1240] [General]
Hi All.

I hope you guys can help. I have an Ender3 V2 (Board version V4.2.2) running the V2.1.3 firmware and I want to upgrade to the new version. I have formatted the uSD correctly and tried many things but no luck. When i insert the SD Card into the Machine and power it on, It still proceeds to the OLD firmware on the device. Im not sure if i should try to update the LCD first and then try again. I also tried to go back to the stock firmware, and got the same outcome. I know everyone will tell me its the SD Card, but i have tried multiple ones NTFS, exFAT, and the required FAT32 (With MBR as mentioned) but I can guarantee its correct. 

Any thoughts?

Dieter

**Comment by mriscoc** [comment]
Change / Format correctly your SD card: https://github.com/mriscoc/Ender3V2S1/wiki/How-to-install-the-firmware that is the main cause of firmware update failures in the 4.2.2 board. Don't forget to rename the firmware filename.

**Comment by Dieter-hamer** [comment]
Hi,

Yes i did do it that way but no luck. Any other thoughts? 

Dieter

**Reply by MarshaJ773** [reply]
Did you use an 8GB sized card or smaller?

**Reply by Dieter-hamer** [reply]
Hi MarshaJ773.

I used only 8gb SD cards, not smaller. I don't have smaller ones.

**Comment by Dieter-hamer** [comment]
Hi,

I used an 8Gb SD card. The one that came with the printer and a second 8gb
SanDisk Ultra class16 .

But none of them worked

Dieter

**Reply by mriscoc** [reply]
8GB are ok, but maybe you need to try with other brands and check the format carefully, also short file names like F1.bin

**Comment by Dieter-hamer** [comment]
Ok I will try it.

Thanks

**Comment by jorymorrison** [comment]
What is the exact filename you are using? If you are using "firmware.bin" try "FIRMWARE.bin".

**Comment by Dieter-hamer** [comment]
I'm using the original file name. So do some of you recommend to change the
name to Firmware1.bin or such?

**Reply by mriscoc** [reply]
That is in the "**how to install**" guide, why don't you follow the instructions from there?

**Reply by Dieter-hamer** [reply]
I apologize, it seems that I have missed that section. I did follow it on the first day about 1.5years ago. I'm sorry I missed that section. I will try to upgrade the firmware again today and let you guys know.

Dieter

**Reply by Dieter-hamer** [reply]
I got it working by renaming the file to FW1.1 or Firmware1.


Thanks for the help everyone who read this post and sorry for wasting your times . 


Greetings
 Dieter
